client/ayon_resolve/api/plugin.py

```diff
@@ -454,6 +456,13 @@ def convert(self):
         else:
             self.tag_data["asset"] = self.ti_name
 
+        # AYON unique identifier
+        folder_path = "/{}/{}".format(
+            self.tag_data["hierarchy"],
+            self.tag_data["asset"],
```

Comment:
Not sure how much work it would be, but this should not be `"asset"` (openpype term) but `"folder_name"`.

Reply:
Yes that is residual OP key name, but we have to keep this backward compatible when it comes to loading already created markers from previouse addon versions. Some clients might need transition time for their already created project markers. Lets clearly comment those backward compatibility changes for future references. 

Here is how some comment looks `# Backwards compatible (Deprecated since 24/06/06)` but @BigRoy might have some better idea on how this should be done. 

BTW, there is a lot of backward compatibility code coments which could be already removed.

Reply:
Oh ! As I flagged in the PR header I didn't put too much attention in making the marker data backward compatible. 
But if that's a thing I'll definitely ensure it is!

Reply:
Just pushed some new commits to ensure these changes remains compatible and can load markers generated from `OpenPypeData`.